### ISSUE TITLE
body request error

### ISSUE THREAD

**Issue by isaacOXIN** [teacat/chaturbate-dvr#72]
body request error: client get: Get "https://chaturbate.com/xxx": read tcp 192.168.0.14:51600->104.16.45.196:443: wsarecv: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond.
[INFO] channel is offline, check again 1 min(s) later

xxx - channel name

**Comment by YamiOdymel** [comment]
It may be related to the network issue, not a bug.

Does it occur often?

**Comment by isaacOXIN** [comment]
Any channel offline

**Comment by YamiOdymel** [comment]
Can you visit chaturbate.com normally?

Does it require you to do an **_identity check_** to watch streams?

**Comment by isaacOXIN** [comment]
Could this be due to incognito or proxy?

**Comment by YamiOdymel** [comment]
Yes, it could be a Proxy or VPN issue. I'm curious if you can browse Chaturbate normally.

I've also noticed that some countries require extra identity checks to view streams.
(which chaturbate-dvr doesn't support yet)

**Comment by YamiOdymel** [comment]
The program cannot fix networking issues. However, if it's related to identity checks (which may be related to #61), it's possible to bypass them by providing [cookies](https://github.com/teacat/chaturbate-dvr#-cookies--user-agent) in [v2.0.0](https://github.com/teacat/chaturbate-dvr/releases/tag/v2.0.0).
